docs/how-to-use-proof.md

```diff
@@ -0,0 +1,31 @@
+# A Guide on How to Use Asset Proofs in Scalar DL
+
+This document sets out some guidelines for using Asset Proofs in Scalar DL.
+
+## What is an Asset Proof in Scalar DL?
+
+An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset record. It is composed of the following items so that it can be treated as evidence of the creation of an asset record by a ledger.
+
+- The ID of an asset record
+- Age of the asset record
+- Nonce of the execution request that creates the asset record
+- A cryptographic hash of the asset record
+- The digital signature of the above four records 
+
+## The Benefits of Asset Proofs
+
+Since Asset Proofs are evidence at the time of execution by a ledger, it is hard for the leger to tamper data after the evidence is created because the proofs and the ledger states would be inconsistent.
+Thus, using and managing Asset Proofs appropriately could reduce the risk of tampering of data, so it could make Scalar DL more tamper-evident without creating additional computational resources such as ordering components.
```

Comment:
What does "using and managing Asset Proofs" mean in this document?
I'm not sure why "managing" is with "using" because their subjects are different.
I think a user or validator can use AssetProofs since Scalar DL manages them.

Reply:
Good point!
From my perspective, the user-side or client-side not only uses proofs but also can manage them like storing them in client-side storage for later use. In contract, the ledger just creates them and doesn't manage them so it doesn't store them, for example. That's what I meant for `manage` to specify clients can/should do more than `use`.
But `manage` might be a too blur word. If you feel it is confusing, maybe we can say like this to imply that client can do more than just `using`. How does it sound?

`using and managing` -> `making use of`

Reply:
Thank you for your reply! It makes sense 👍 
`making use of` looks better.

Reply:
OK! I updated it with 6163a61.